mirrord/agent/src/cli.rs

```diff
@@ -35,6 +35,11 @@ pub struct Args {
     /// Pause the target container while clients are connected.
     #[arg(short = 'p', long, default_value_t = false)]
     pub pause: bool,
+
+    /// Return an error after accepting the first client connection, in order to test agent error
+    /// cleanup.
+    #[arg(long, default_value_t = false, hide = true)]
+    pub test_error: bool,
```

Comment:
I like `test_error` better, because what it does is to return an error for testing purposes, and the cleanup part is helpful background about what we actually do with it currently, but not really part of the flag's effect. So in my opinion it should be at most part of the documentation.

But if you think it would make it clearer, `test_error_cleanup` would also be fine with me.

Reply:
The name is fine as-is for me as well.